### PR TITLE
Add css colors `ju-comment` and `ju-judoc`

### PR DESCRIPTION
**Pull request by janmasrovira**
This pr adds colors for comments and judoc for the html backend

### CODE REVIEW

File: assets/css/source-ayu-light.css [hunk 1]
```diff
@@ -31,6 +31,14 @@ body {
     color: #787b80;
 }
 
+.ju-comment {
+    color: #ba919966;
+}
```
Comment (jonaprieto) [inline assets/css/source-ayu-light.css:36]: Keep just one.

**Comment by jonaprieto** [comment]
Idea. Perhaps we can think about generating these color schemes automatically.
They are simple enough, and during the process, we can make sure the CSS
stylesheets define all the colors we need, one for each face.  

Also, we could offer some customisation, perhaps by exposing
some optional fields in the package Yaml to redefine the default
colors and other info, like the title or footer.
For example, we could do something like:

```yaml
# juvix.yaml
...
html:
  theme: dark
  colors:
  - comment: #c3c3c3
  - delimiter: #F3f3F3
  ...
...
```

File: assets/css/source-ayu-light.css (hunk 1 above)
Comment (janmasrovira) [reply assets/css/source-ayu-light.css:36]: solved

**Comment by janmasrovira** [comment]
> Idea. Perhaps we can think about generating these color schemes automatically. They are simple enough, and during the process, we can make sure the CSS stylesheets define all the colors we need, one for each face.

We can explore that, as it would offer more safety.

> Also, we could offer some customisation, perhaps by exposing some optional fields in the package Yaml to redefine the default colors and other info, like the title or footer. For example, we could do something like:
> 
> ```yaml
> # juvix.yaml
> ...
> html:
>   theme: dark
>   colors:
>   - comment: #c3c3c3
>   - delimiter: #F3f3F3
>   ...
> ...
> ```

I'm not 100% convinced about using the `juvix.yaml` file for that. Maybe we can offer the user to pass a custom css file as an argument to the cli. The `juvix.yaml` file could then have an optional field that points to that custom css file.
Either way, let's discuss this in a separate issue.

**Review by jonaprieto — APPROVED** [review]

**Comment by jonaprieto** [comment]
I'll open an issue to continue this discussion later.

> using the `juvix.yaml` file for that.

Why not? this usage example is a particular case, the way I see it, of providing specific parameters/input to the backend target for the build process.

**Pull request merged**